### DISCUSSION TITLE
small icons doesnt work and taskbar apps is cramped

### DISCUSSION THREAD

**Discussion by patchie** [valinet/ExplorerPatcher#1960] [Q&A] [answered]
There is something strange with the setup.

Questions:
1. How can i make more space horizontally for the apps in the taskbar when its set to "never combine"?
You can see below that the taskbar has a huge area that is not utilized in the center.
![image](https://github.com/valinet/ExplorerPatcher/assets/1792727/967b19c3-cff4-48f8-ba5e-ccdcd807a833)

2. How can i get the old "right-click" menu on the taskbar? So that i can add a folder with shortcuts to the taskbar(win2k style). Works in win10.
![image](https://github.com/valinet/ExplorerPatcher/assets/1792727/be0ca2ee-36de-489d-994f-fb603ded33d9)

3. Is it possible to make the taskbar higher, so that i can have several rows of taskbar apps there, like in win10?
Something like my example screenshot below:
![image](https://github.com/valinet/ExplorerPatcher/assets/1792727/2a61a3ab-0f1c-41dd-97b8-4d22a5f019ae)

**Comment by aldeniangman** [comment]
it is a bug

**Comment by Poopooracoocoo** [comment]
It looks like your issue is that for some reason it's not using the Windows 10 taskbar. I'm able to use the Windows 10 taskbar on Windows 11 build 22621.2134 with some issues due to the KB5029263 cumulative update. Note that EP does not support Insider Preview builds and preview updates.

**Comment by patchie** [comment, marked as answer]
This was fixed in a later version btw!

Thanks a lot!
